src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Display from "./Display";
+import ButtonPanel from "./ButtonPanel";
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <div id="calculator">
+        <Display></Display>
```

Comment:
These should be:

```javascript
<Display />
<ButtonPanel />
```
Also let us pass to `Display` a `result` prop so that we can test if it is rendering correctly in the browser. For now we can set the `result` prop to any string. Later on we will change it to a more meaningful value.